server/reports/commission/commission-endpoint.js

```diff
@@ -0,0 +1,110 @@
+//module "commission-report"
+import json2csvExports from "json2csv";
+import {CommissionReport} from "./commission.js";
+import MindbodyAccess from "../../api-manager.js";
+
+/*a handy request URLS for testing:
+http://localhost:8080/reports/commission?format=csv&startdate=2020-03-01T00:00:00&enddate=2020-03-031T00:00:00
+*/
+export function handleCommissionRptRequest(request, response) {
+  //console.log("handleCommissionRptRequest:");
+  //console.log("request: " + request.baseUrl + " - " + JSON.stringify(request.query));
+
+  //validate input parameters
+  //query string looks like: ?format=[json|csv]&startdate=[whatever]&enddate=[whatever]
+  let format = request.query.format;
+  if (!format) {
+    response.send("Missing \"format\" parameter.");
+    return;
+  }
+  if (!(format === "json" || format === "csv")) {
+    response.send("Bad \"format]\" parameter. Must be \"json\" or \"csv\"");
+    return;
+  }
+
+  let startDate = request.query.startdate;
+  if (!startDate) {
+    response.send("Missing \"startdate\" parameter.");
+    return;
+  }
+
+  let endDate = request.query.enddate;
+  if (!endDate) {
+    response.send("Missing \"enddate\" parameter.");
+    return;
+  }
+
+
+  //pass validated input params to report generator
+  let rptGenerator = new CommissionReport();
+  rptGenerator.setStartDate(startDate);
+  //rptGenerator.setEndDate(endDate);
```

Comment:
endDate isn't hooked up and tested yet but there's a trello ticket for this so it's fine:
https://trello.com/c/Sb38FnmQ/76-implement-date-ranges